etc/scripts/docenizerJava.sh

```diff
@@ -0,0 +1,5 @@
+#/usr/bin/bash
+
+curl https://docs.oracle.com/javase/specs/jvms/se16/html/jvms-6.html -o `pwd`/vendor/jvms.html
```

Comment:
A question that's me being ignorant more than anything: Does this skip the download if it's already there?

Reply:
Did not do that previously, updated to only download the file if necessary.